static/bidder-info/dxkulture.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://ads.kulture.media/pbs"
+maintainer:
+  email: "devops@kulture.media"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  redirect:
+    url: "https://ads.kulture.media/usync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+    userMacro: "$UID"
```

Comment:
verified usersync redirect url is working

https://ads.kulture.media/usync?gdpr=1&gdpr_consent=1&us_privacy=1&cb=https://ib.adnxs.com/setuid?bidder=kulture&gdpr=&gdpr_consent=&gpp=&gpp_sid=&f=b&uid=$UID